src/contracts/facilitators/aave/misc/GhoManager.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+import {IPoolConfigurator} from '@aave/core-v3/contracts/interfaces/IPoolConfigurator.sol';
+import {IGhoVariableDebtToken} from 'src/contracts/facilitators/aave/tokens/interfaces/IGhoVariableDebtToken.sol';
+
+/**
+ * @title GhoManager
+ * @author Aave
+ * @notice Helper contract for managing key risk parameters of the GHO reserve within the Aave Facilitator
+ * @dev This contract is intended to be granted as PoolAdmin
+ */
+contract GhoManager is Ownable {
+  /**
+   * @dev Constructor.
+   * @param owner The owner address of this contract
+   */
+  constructor(address owner) {
+    transferOwnership(owner);
```

Comment:
No need to transfer the owner at deployment time. I d remove the whole constructor

Reply:
updated in `89c8f02f33d6deca6392c879a01a8976940d1376`